src/ert/_c_wrappers/job_queue/driver.py

```diff
@@ -42,7 +44,9 @@ class Driver(BaseCClass):
     _set_max_running = ResPrototype("void queue_driver_set_max_running( driver , int)")
     _get_name = ResPrototype("char* queue_driver_get_name( driver )")
 
-    def __init__(self, driver_type, max_running=1, options=None):
+    def __init__(
+        self, driver_type, max_running: int = 1, options: List[Tuple[str, str]] = None
```

Comment:
won't driver_type get at least `Any`?

Reply:
I can try to figure out the correct type for this.

Reply:
`QueueDriverEnum`